packages/core/deploy/015_deploy_sink_governor.js

```diff
@@ -13,5 +13,5 @@ const func = async function (hre) {
   });
 };
 module.exports = func;
-func.tags = ["SinkGovernor", "sink-governor"];
-func.dependencies = ["Finder"];
+func.tags = ["SinkGovernor", "bridge-l2"];
```

Comment:
These tags make sense. Would you mind adding them to some sort of readme?

hardhat-deploy tags:
- `dvm`: deploys core voting system infrastructure
- `bridge-l2`: deploys infrastructure to L2 to enable cross-EVM communication.
- `bridge-l1`: deploys infrastructure to L1 to enable cross-EVM communication

Reply:
Do we have a readme for hardhat deployment at all?

Reply:
Sounds good to me